modules/06 Angular/Assignments/12/answer12/src/app/image/image.component.html

```diff
@@ -0,0 +1,6 @@
+<div *ngFor="let image of images">
+  <a href="{{image.url}}" target="_blank">
```

Comment:
The component says it's an image component, but I'm seeing a link. Try not to violate SRP from SOLID